<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLogin</class>
 <widget class="QDialog" name="DialogLogin">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>332</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>22</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>登录界面</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>361</width>
     <height>126</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>10</number>
    </property>
    <property name="spacing">
     <number>12</number>
    </property>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtUserName">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 0, 0)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="pwdLogin">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 0, 0)</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labUserName">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labPassword">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string>密  码：</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>pwdLogin</zorder>
   <zorder>txtUserName</zorder>
   <zorder>labPassword</zorder>
   <zorder>labUserName</zorder>
  </widget>
  <widget class="QWidget" name="layoutWidget_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>361</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnLogin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>219</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widgetTitle" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="labTitle">
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>用户名登陆</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>widgetTitle</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnLogin</sender>
   <signal>clicked()</signal>
   <receiver>DialogLogin</receiver>
   <slot>judgeUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogLogin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>judgeUser()</slot>
 </slots>
</ui>
